src/Mod/PartDesign/App/FeatureSketchBased.cpp

```diff
@@ -543,6 +543,18 @@ void ProfileBased::getUpToFace(TopoDS_Face& upToFace,
     }
 }
 
+double ProfileBased::getThroughAllLength() {
+    TopoDS_Shape profileshape;
+    TopoDS_Shape base;
+    profileshape = getVerifiedFace();
+    base = getBaseShape();
+    Bnd_Box box;
+    BRepBndLib::Add(base, box);
+    BRepBndLib::Add(profileshape, box);
+    box.SetGap(0.0);
+    return 1.01*sqrt(box.SquareExtent());

```

Comment:
I miss documentation on what you do why.
- why the magic number 1.01? I guess to be a bit longer than necessary, so a comment would be helpful or others
- why do you take the root of the square extent and not the extent itself?

Reply:
Comment added.